vendor.conf

```diff
@@ -104,7 +104,7 @@ github.com/containerd/containerd 3addd840653146c90a254301d6c3a663c7fd6429
 github.com/tonistiigi/fifo 1405643975692217d6720f8b54aeee1bf2cd5cf4
 
 # cluster
-github.com/docker/swarmkit eb07af52aa2216100cff1ad0b13df48daa8914bf
+github.com/docker/swarmkit 85019dae9bacf2b68b81f092c34e755e2ee76b02
```

Comment:
Hey @RenaudWasTaken 👋 

You should be able to rebase and drop the swarmkit re-vendor now that a newer version is in moby via https://github.com/moby/moby/pull/33642.